packages/destination-actions/src/destinations/fullstory/request-params.ts

```diff
@@ -139,3 +139,23 @@ export const deleteUserRequestParams = (settings: Settings, userId: string): Req
     }
   }
 }
+
+/**
+ * Returns {@link RequestParams} for the V2 Create User HTTP API endpoint.
+ *
+ * @param settings Settings configured for the cloud mode destination.
+ * @param requestBody The request body containing user properties to set.
+ */
+
+export const createUserRequestParams = (settings: Settings, requestBody: Object): RequestParams => {
+  const defaultParams = defaultRequestParams(settings, `v2beta/users?${integrationSourceQueryParam}`)
```

Comment:
You are correct. @sabrina-li is moving integration source to request headers. Since extra headers would just be ignored. I can add this new header and it will be picked up when ready.

Reply:
segment is special in that, if the `integration` is not specified (either in request or in header), the events api always attempt to check for `User-Agent` as well. With v2 we plan on removing the query param completely before GA so only header will be supported soon.
Also, I don't think the users endpoint has integration source (yet): there's no where to put it since it doesn't result in event bundles like the events api.
Anyways, I vote for putting integrations source to headers or both(req&header), it shouldn't hurt.